docs/src/main/asciidoc/_high-availability/load-balancing/Affinity_Cookie.adoc

```diff
@@ -0,0 +1,44 @@
+[[load-balancer-affinity-cookie]]
+= Support for load-balancers relying on affinity cookie
+
+While the Apache family of load-balancers relies on attaching session affinity (routing) information to a `JSESSIONID`,
```

Comment:
"to a `JSESSIONID` _cookie or a `jsessionid` path parameter_"

Reply:
Added, thanks.